zmon-controller-app/src/main/resources/config/application.yml

```diff
@@ -26,7 +26,7 @@ server:
 
 zmon:
     jwt:
-      secret: ${JWT_SECRET}
+      secret: ${ZMON_JWT_SECRET}
```

Comment:
upps, sorry, did not see your env var here --- now we don't need anything in the YAML as `zmon.jwt.secret` is the same as `ZMON_JWT_SECRET` :smirk: